accounts-db/src/buffered_reader.rs

```diff
@@ -109,11 +98,11 @@ pub(crate) trait RequiredLenBufFileRead<'a>: RequiredLenBufRead + FileBufRead<'a
 impl<'a, T: RequiredLenBufRead + FileBufRead<'a>> RequiredLenBufFileRead<'a> for T {}
 
 /// read a file a large buffer at a time and provide access to a slice in that buffer
-pub struct BufferedReader<'a, T> {
+pub struct BufferedReader<'a, const N: usize> {
```

Comment:
maybe the reader should be renamed to `StackBufferedReader` or something like that? wdyt?